rasa/core/nlg/interpolator.py

```diff
@@ -36,14 +38,18 @@ def interpolate_text(template: Text, values: Dict[Text, Text]) -> Text:
 
         return text
     except KeyError as e:
-        logger.exception(
-            f"Failed to fill utterance template '{template}'. "
-            f"Tried to replace '{e.args[0]}' but could not find "
-            f"a value for it. There is no slot with this "
-            f"name nor did you pass the value explicitly "
-            f"when calling the template. Return template "
-            f"without filling the template. "
-        )
+        # we assume that if the value of the {key: value} pair starts with `/`,
```

Comment:
why do we interpolate it then in the first place?

Reply:
Because we interpolate anything that has the curly braces in it! It's just an unfortunate thing about the regex format versus the slot interpolation format.  

We could try to make a nicer regex for this, but as far as I could see in the rest of the code, we don't have a regex for the /intent{"entity": "value"} format, we just pass things that that with `/` to the regexinterpreter. Though I guess that interpreter does have a regex for processing 🤔 

Basically one thing we could have is a button payload that is e.g. `/intent{"entity": {slot_name}}` and we want it to correctly fill to `/intent{"entity": "value"}` without yelling that it can't find a slot called `"entity": {slot_name}`. Any suggestions for improving that?

Reply:
Thanks for the explanation! Can't we restrict the interpolation to anything which **does not** start with `/`?

Reply:
That's what I thought first, too. But @erohmensing has a point - there could be use cases to set an entity/slot to the value of a different slot, maybe while confirming data from a backend inquiry or similar. So hiding the exception seem to be the way to go for me.

Reply:
According to doc you should use double curly braces (standard f-string escaping for curly braces) if you want rasa NOT TO replace (interpolate) the alleged key words. You can still use curly braces inside, like

`/intent{{"entity": "{my_slot}"}}`

even though it gets harder to read. I would recommend putting that in the docs and close the PR.

Reply:
@erohmensing What do you think?